docs/source/tutorials/quickstart.rst

```diff
@@ -2,145 +2,150 @@
+ information on using manim with Jupyterlab or Jupyter notebook, go to the
+ documentation for the
+ :meth:`IPython magic command <manim.utils.ipython_magic.ManimMagic.manim>`,
+ ``%%manim``.
 
+Overview
+*************************************************
 
-Start a new project
-*******************
+This quickstart guide will lead you through creating a sample project using manim: an animation
+engine for precise programmatic animations.
 
-To start a new manim video project, all you need to do is choose a single
-folder where all of the files related to the video will reside.  For this
-example, this folder will be called ``project``,
+First, you will use a command line
+interface to create a ``Scene``, the class through which manim generates videos.
+In the ``Scene`` you will animate a circle. Then you will add another ``Scene`` showing
+a square transforming into a circle. This will be your introduction to manim's animation ability.
+Afterwards, you will position multiple manim objects (mobjects). Finally, you
+will learn the syntax of ``.animate``, a powerful feature that displays how
+mobjects change when you modify them with methods.
 
-.. code-block:: bash
 
-   project/
+Starting a new project
+*************************************************
 
-Every file containing code that produces a video with manim will be stored
-here, as well as any output files that manim produces and configuration files
-that manim needs.
+Start by creating a new folder. For the purposes of this guide, name the folder ``project``:
 
-.. note::
+.. code-block:: bash
 
-   In case you like to work with Jupyterlab / Jupyter notebooks, there is good news:
-   Manim ships with a ``%%manim`` IPython magic command which makes it easy to use
-   in such a setting as well. Find out more in the
-   :meth:`corresponding documentation <manim.utils.ipython_magic.ManimMagic.manim>`.
+   project/
+
+This folder is the root folder for your project. It contains all the files that manim needs to function,
+as well as any output that your project produces.
 
 
-Your first Scene
-****************
+Animating a circle
+*************************************************
 
-To produce your first scene, create a new file in your project folder called
-``scene.py``,
+1. Create a new file in your project folder named ``scene.py``.
 
 .. code-block:: bash
 
    project/
    └─scene.py
 
-and copy the following code in it.
+2. Copy the following code into ``scene.py``:
 
 .. code-block:: python
 
    from manim import *
 
 
-   class SquareToCircle(Scene):
+   class CreateCircle(Scene):
        def construct(self):
            circle = Circle()  # create a circle
            circle.set_fill(PINK, opacity=0.5)  # set the color and transparency
            self.play(Create(circle))  # show the circle on screen
 
-Then open your command line, navigate to your project directory, and execute
+3. Open the command line, navigate to your project folder, and execute
```

Comment:
Again, many new users would struggle at this step.

Reply:
I could expand on getting to the command line for Windows, but I don't own a mac or have Linux running on my computer, so I wouldn't be able to cover potential differences between operating systems. What would you recommend here?

Reply:
Hmm, I think you can leave it for now. Someone should revisit this in the future though.